api/services/ChatService.js

```diff
@@ -38,6 +36,9 @@ function addVideoMessage(message) {
 }
 
 function addMessage(chat) {
-  chats.push(chat);
-  sails.io.sockets.in('chatting').emit('chat', chat);
+  Chat.create(chat)
+    .then((c) => {
+      sails.io.sockets.in('chatting').emit('chat', c);
+    })
+    .catch((e) => console.log(e));
```

Comment:
Can you log this using log4js to get some additional logging appenders? Just in case we pipe logs somewhere else also.